packages/playground/src/components/node_selection.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <v-card color="primary" class="d-flex mt-3 pa-3">
+    <v-card-title class="pa-0">Node Selection</v-card-title>
+  </v-card>
+  <h6 class="text-h6 mb-4 mt-2 ml-2">Choose a way to select Node</h6>
+  <v-radio-group v-model="selection" color="primary" inline>
+    <v-radio label="Automated" value="automated"></v-radio>
+    <v-radio label="Manual" value="manual"></v-radio>
+  </v-radio-group>
+</template>
+
+<script lang="ts" setup>
+import { ref, watch } from "vue";
+
+const emits = defineEmits({
+  "update:model-value": (value: Selection) => true || value,
+});
+
+enum Selection {
+  AUTOMATED = "automated",
+  MANUAL = "manual",
+}
```

Comment:
We can remove this to another file and export it to use the enum in selectFarm condition instead of using a string.